.github/workflows/examples.yaml

```diff
@@ -18,7 +18,7 @@ on:
 env:
   EMSDK_VERSION: "3.1.40"
   TINYGO_VERSION: "0.35.0"
-  ZIG_VERSION: "0.11.0"
```

Comment:
I was having some issues in CI with this and it was easier to bump to the latest which I have installed to match it up